### PR TITLE
Creat advice css

### PR DESCRIPTION
**Pull request by MontherIsmail**
Relates #20 
Making style of header, sections and next button

### CODE REVIEW

**Review by AbdallahGot — CHANGES_REQUESTED** [review]
image

try to improve these parts style.

File: advice/advice.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+}
+body{
+    font-family: 'Montserrat', sans-serif;
+    background-color: var(--secondary-color);
+    color: var(--fonts-color);
+}
+.header{
+    display: flex;
+    /*justify-content: space-between;*/
```
Comment (Ibrahim-Jarada) [inline advice/advice.css:25]: Remove the comment code line

File: advice/advice.html [hunk 2]
```diff
@@ -1,11 +1,48 @@
...
-    <title>advice</title>
-  </head>
-  <body></body>
-</html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:8]: Remove one line from the repeated ones

File: advice/advice.html [hunk 3]
```diff
@@ -1,11 +1,48 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./advice.css">
+    <title>Document</title>
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:12]: Are you sure that's the right title?

File: advice/advice.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+    /*************** Back Icon *********************/
+         function backIcon(){
+             window.location.href ="../symptoms/Symptoms.html";
+         }
+
+    /***************Dark Mode***********************/
+        var icon = document.getElementById("icon");
```
Comment (Ibrahim-Jarada) [inline advice/advice.js:7]: Try to avoid using var as you can, its preferable to use const here instead of var

File: advice/advice.html (hunk 3 above)
Comment (MontherIsmail) [reply advice/advice.html]: Done

**Review by NadaSaleh20 — DISMISSED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

**Review by Rawanzaqout — APPROVED** [review]

File: advice/advice.html [hunk 5]
```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
         <img src="../assets/dark-mode.png" class="dark-mode-icon" id="icon">
     </header>
     <section class="container">
-        <article class="part-1">
+        <artical class="cont">
+
             <section class="part">
-                <img src="../assets/01.png" class="img-adv">
+                <img src="../images/01.png" class="img-adv">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:26]: Why you changed the folder from assets to images? Are you sure that you have images folder in your branch?
Are you sure that you solved the conflict in a right way?

File: advice/advice.html [hunk 6]
```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
...
                 <h1 class="title">Wear A Mask</h1>
             </section>
+
             <section class="part">
-                <img src="../assets/02.png" class="img-adv">
+                <img src="../images/02.png" class="img-adv">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:31]: Same as above

File: advice/advice.html [hunk 7]
```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
...
                 <h1 class="title">Wash Your Hand</h1>
             </section>
+
             <section class="part">
-                <img src="../assets/03.png" class="img-adv">
+                <img src="../images/03.png" class="img-adv">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:36]: Same as above

File: advice/advice.html [hunk 8]
```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
...
                 <h1 class="title">Consult Doctor</h1>
             </section>
-        </article>
-        <article class="part-2">
+
             <section class="part">
-                <img src="../assets/04.png" class="img-adv">
+                <img src="../images/04.png" class="img-adv">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:41]: Same as above

File: advice/advice.html [hunk 9]
```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
...
                 <h1 class="title">Avoid Touching</h1>
             </section>
+
             <section class="part">
-                <img src="../assets/05.png" class="img-adv">
+                <img src="../images/05.png" class="img-adv">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:46]: Same as above

File: advice/advice.html [hunk 10]
```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
...
                 <h1 class="title">Avoid Contact</h1>
             </section>
+
             <section class="part">
-                <img src="../assets/06.png" class="img-adv">
+                <img src="../images/06.png" class="img-adv">
```
Comment (Ibrahim-Jarada) [inline advice/advice.html:51]: Same as above

**Review by Ibrahim-Jarada — CHANGES_REQUESTED** [review]
Be careful! when you solve conflicts!

**Review by AbdallahGot — APPROVED** [review]

**Review by lina-jamal — APPROVED** [review]

File: advice/advice.html (hunk 10 above)
Comment (MontherIsmail) [reply advice/advice.html:51]: Done

**Pull request merged**